.github/workflows/PrBot.yml

```diff
@@ -0,0 +1,60 @@
+name: Bot scan
+on:
+  pull_request:
+    types: [ opened, reopened ]
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  Snyk_scanning:
+    permissions: write-all
+    name: Snyk Bot scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Snyk Bot scan
+        continue-on-error: true
+        run: |
+         echo "----------Download Latest Snyk CLI-----------"
+             npm i -g snyk@latest
```

Comment:
This cannot possibly work, since we use pnpm on our repo.
You don't need to install snyk. You can use `npx` which will download and execute it without polluting the lock file and node_modules.